dockers/docker-ptf/Dockerfile.j2

```diff
@@ -168,6 +171,16 @@ RUN rm -rf /debs \
     && cd /opt             \
     && wget https://raw.githubusercontent.com/p4lang/ptf/master/ptf_nn/ptf_nn_agent.py
 
+{% if PTF_ENV_PY_VER == "py3" %}
+RUN git clone https://github.com/robot527/tac_plus \
+    && cd tac_plus \
+    && cd tacacs+  \
+    && ./configure \
+    && make install \
+    && ln -s /usr/local/sbin/tac_plus /usr/sbin/tac_plus \
+    && ln -s /usr/local/bin/tac_pwd /usr/sbin/tac_pwd
+{% endif %}
```

Comment:
Why not use the source code from an organization like https://github.com/facebook/tac_plus
I assume it would be better than the fork of a personal account?

Reply:
@wangxin thanks. I have updated the repo.